### PR TITLE
sed: add German translation

### PR DESCRIPTION
**Pull request by DoPaNik**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/osx/sed.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{dateiname}}`
+
+- Ersetzt das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:
+
+`sed 's#{{finde}}#{{ersetze}}#' {{dateiname}}`
+
+- Löscht die Zeile an der bestimmten Zeilennummer in einer Datei:
+
+`sed -i '' '{{zeilennummer}}d' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:36]:
```suggestion
# sed

> Text skriptfähig bearbeiten.
> Siehe auch: `awk`, `ed`.
> Weitere Informationen: <https://keith.github.io/xcode-man-pages/sed.1.html>.

- Ersetzen Sie alle Vorkommen von `apple` (Basic Regex) durch `mango` (Basic Regex) in allen Eingabezeilen und geben Sie das Ergebnis in `stdout` aus:

`{{Befehl}} | sed 's/Apfel/Mango/g''

- Führen Sie eine bestimmte Skript-[Datei] aus und geben Sie das Ergebnis in `stdout` aus:

`{{Befehl}} | sed -f {{Pfad/zu/script.sed}}`

- Ersetzen Sie alle Vorkommen von `apple` (erweiterte Regex) durch `APPLE` (erweiterte Regex) in allen Eingabezeilen und geben Sie das Ergebnis in `stdout` aus:

`{{Befehl}} | sed -E 's/(Apfel)/\U\1/g''

- Gib nur eine erste Zeile nach `stdout` aus:

`{{Befehl}} | sed -n '1p'`

- Ersetzen Sie alle Vorkommen von `apple` (einfacher Regex) durch `mango` (einfacher Regex) in einer `Datei` und speichern Sie eine Sicherungskopie des Originals in `file.bak`:

`sed -i bak 's/apple/mango/g' {{path/to/file}}`
```

Optional fix: this page can be synced with the [original](https://github.com/tldr-pages/tldr/blob/main/pages/osx/sed.md) one.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

**Review by kbdharun — APPROVED** [review]
LGTM, after @EmilySeville7cfg's suggestion.

File: pages.de/osx/sed.md (hunk 1 above)
Comment (sbrl) [reply pages.de/osx/sed.md:36]: I don't understand what you're asking for here @EmilySeville7cfg?

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

File: pages.de/osx/sed.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages.de/osx/sed.md:36]: I think I've asked to replace some translated placeholders to original ones where replacement occur. As this fix is optional and is broken (path placeholder is not translated) I can mark this conversation as resolved.

File: pages.de/osx/sed.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# sed
+
+> Führe grundlegende Texttransformationen an der Eingabe aus.
+> Weitere Informationen: <https://ss64.com/osx/sed.html>.
+
+- Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:
+
+`sed 's/{{finde}}/{{ersetze}}/' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:8]:
```suggestion
- Ersetzt das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:

`sed 's/{{finde}}/{{ersetze}}/' {{Pfad/zu/Datei}}`
```

Required fix: [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): `path/to/*` translated generic placeholder expected.

File: pages.de/osx/sed.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetzt alle Vorkommen eines erweiterten regulären Ausdrucks in einer Datei:
+
+`sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:12]:
```suggestion
- Ersetzt alle Vorkommen eines erweiterten regulären Ausdrucks in einer Datei:

`sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{Pfad/zu/Datei}}`
```

The same issue as above.

File: pages.de/osx/sed.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetzt alle Vorkommen einer Zeichenkette [i]n einer Datei, wobei die Datei überschrieben wird (d.h. an Ort und Stelle):
+
+`sed -i '' 's/{{finde}}/{{ersetze}}/g' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:16]:
```suggestion
- Ersetzt alle Vorkommen einer Zeichenkette [i]n einer Datei, wobei die Datei überschrieben wird (d.h. an Ort und Stelle):

`sed -i '' 's/{{finde}}/{{ersetze}}/g' {{Pfad/zu/Datei}}`
```

File: pages.de/osx/sed.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetzt nur die Zeilen, die dem Zeilenmuster entsprechen:
+
+`sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:20]:
```suggestion
- Ersetzt nur die Zeilen, die dem Zeilenmuster entsprechen:

`sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{Pfad/zu/Datei}}`
```

The same issue as above.

File: pages.de/osx/sed.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Druckt nur den Text zwischen der n-ten Zeile bis zur nächsten Leerzeile:
+
+`sed -n '{{zeilennummer}},/^$/p' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:24]:
```suggestion
- Druckt nur den Text zwischen der n-ten Zeile bis zur nächsten Leerzeile:

`sed -n '{{zeilennummer}},/^$/p' {{Pfad/zu/Datei}}`
```

The same issue as above.

File: pages.de/osx/sed.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Mehrere Ausdrücke zum Suchen und Ersetzen auf eine Datei anwenden:
+
+`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:28]:
```suggestion
- Mehrere Ausdrücke zum Suchen und Ersetzen auf eine Datei anwenden:

`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{Pfad/zu/Datei}}`
```

The same issue as above.

File: pages.de/osx/sed.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetzt das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:
+
+`sed 's#{{finde}}#{{ersetze}}#' {{dateiname}}`
```
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:32]:
```suggestion
- Ersetzt das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:

`sed 's#{{finde}}#{{ersetze}}#' {{Pfad/zu/Datei}}`
```

The same issue as above.

File: pages.de/osx/sed.md (hunk 1 above)
Comment (EmilySeville7cfg) [inline pages.de/osx/sed.md:36]:
```suggestion
- Löscht die Zeile an der bestimmten Zeilennummer in einer Datei:

`sed -i '' '{{zeilennummer}}d' {{Pfad/zu/Datei}}`
```

The same issue as above.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
Required fixes found, changes requested.
Thanks for a contribution! 🚀

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Could you please adopt the changes from 3c944f84c349d515aac753d821bd88adbba4026c and use imperative mood in the example descriptions, like the contributing guidelines suggest?

**Comment by DoPaNik** [comment]
> 

Done ;)  Thanks for that hint

**Comment by DoPaNik** [comment]
> Required fixes found, changes requested. Thanks for a contribution! 🚀

Thanks!

File: pages.de/osx/sed.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+# sed
+
+> Führe grundlegende Texttransformationen an der Eingabe aus.
+> Weitere Informationen: <https://ss64.com/osx/sed.html>.
+
+- Ersetze das erste Vorkommen einer Zeichenkette in einer Datei und gibt das Ergebnis aus:
+
+`sed 's/{{finde}}/{{ersetze}}/' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:8]:
```suggestion
`sed 's/{{finde}}/{{ersetze}}/' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetze alle Vorkommen eines erweiterten regulären Ausdrucks in einer Datei:
+
+`sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:12]:
```suggestion
`sed -E 's/{{regulärer_ausdruck}}/{{ersetze}}/g' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetze alle Vorkommen einer Zeichenkette [i]n einer Datei, wobei die Datei überschrieben wird (d.h. an Ort und Stelle):
+
+`sed -i '' 's/{{finde}}/{{ersetze}}/g' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:16]:
```suggestion
`sed -i '' 's/{{finde}}/{{ersetze}}/g' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetze nur die Zeilen, die dem Zeilenmuster entsprechen:
+
+`sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:20]:
```suggestion
`sed '/{{zeilenmuster}}/s/{{finde}}/{{resetze}}/' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+- Druck nur den Text zwischen der n-ten Zeile bis zur nächsten Leerzeile:
+
+`sed -n '{{zeilennummer}},/^$/p' {{Pfad/zu/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:24]:
```suggestion
`sed -n '{{zeilennummer}},/^$/p' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+- Suche und Ersetzen mehrere Ausdrücke in einer Datei:
+
+`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:28]:
```suggestion
`sed -e 's/{{finde}}/{{ersetze}}/' -e 's/{{finde}}/{{ersetze}}/' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+- Ersetze das Trennzeichen `/` durch jedes andere Zeichen, das nicht in den Such- oder Ersetzungsmustern verwendet wird, z. B. `#`:
+
+`sed 's#{{finde}}#{{ersetze}}#' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:32]:
```suggestion
`sed 's#{{finde}}#{{ersetze}}#' {{pfad/zu/datei}}`
```

File: pages.de/osx/sed.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+
+- Lösche die Zeile an der bestimmten Zeilennummer in einer Datei:
+
+`sed -i '' '{{zeilennummer}}d' {{Pfad/zur/Datei}}`
```
Comment (pixelcmtd) [inline pages.de/osx/sed.md:36]:
```suggestion
`sed -i '' '{{zeilennummer}}d' {{pfad/zu/datei}}`
```

**Review by pixelcmtd — COMMENTED** [review]
That is precisely why I don't review PRs in languages that I do not speak.

Also, I asked you to update your PR with recent changes to the English page. Could you please do that?

**Comment by DoPaNik** [comment]
Sorry I overseen it - yes,I will adopt that asap

**Pull request closed without merging**